src/components/Photoalbum.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Photos from './Photos';
+
+//destructuring is more informative other tthan just leaving 
+function Photoalbum({ title, photos}) {
+  return (
+    <section>
+      <Photos />
```

Comment:
```suggestion
      <Photos photos={cities}/>
```
Now you can pass the city photos down to the `<Photos>` component.